src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -979,6 +979,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
     {
         _terminal->WritePastedText(hstr);
         _terminal->ClearSelection();

```

Comment:
aside: `Terminal::ClearSelection` should probably just trigger selection, right?

Reply:
Yeah, I'm thinking we may want to move all of the `TriggerSelection()` calls down to `TerminalSelection.cpp`. There's a few times we trigger something off the renderer in `Terminal.cpp`, so we'd basically do the same.